packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -1419,16 +1408,16 @@ describe('Picker', function () {
 
       let picker = getByRole('button');
       expect(picker).toHaveTextContent('Select an option…');
-  
-      let hiddenLabel = getAllByText('Test')[1];
+
+      let hiddenLabel = getByText('Test', {hidden: true, selector: 'label'});
       expect(hiddenLabel.tagName).toBe('LABEL');
       expect(hiddenLabel.parentElement).toHaveAttribute('aria-hidden', 'true');
-      
-      let hiddenSelect = getByRole('listbox');
+
+      let hiddenSelect = getByRole('combobox', {hidden: true}); // not sure why i can't use listbox https://github.com/A11yance/aria-query#elements-to-roles
```

Comment:
Maybe we look into this one a little more? Seems problematic.

Reply:
fixed and added a nice big comment explaining what happened